capstone/src/main/resources/static/index.html

```diff
@@ -52,12 +62,6 @@ <h1>Events</h1>
           <label for="vistor">Visitors Allowed</label>
           <input type="checkbox" id="vistor" name="vistor" value="vistor">
         </div>
-
-        <label for="sort">SORT BY</label>
-        <select name="sort" id="sort" class="button smallDefaultButton">
-            <option value="descending">NEWEST</option>
-            <option value="ascending">OLDEST</option>
-        </select>
```

Comment:
I agree the sort by time is not really useful because people usually want to look at the most recent events.